src/submit.rs

```diff
@@ -179,6 +179,30 @@ impl<'a> Submitter<'a> {
         .map(drop)
     }
 
+    /// Registers an empty file table of nr_files number of file descriptors. The sparse variant is
+    /// available in kernels 5.19 and later.
+    ///
+    /// Registering a file table is a prerequisite for using any request that
+    /// uses direct descriptors.
+    pub fn register_files_sparse(&self, nr: u32) -> io::Result<()> {
```

Comment:
As a new feature that has not been tested on ci, it needs to be marked unstable.

Reply:
Will do. Should I fix, squash, and submit a new branch?

Reply:
You can push directly to current branch